File: CabinRent/CabinRent.Model/Klijent.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CabinRent.Model
{
    public partial class Klijent
    {
       
        public int KlijentId { get; set; }
        public string Ime { get; set; } 
        public string Prezime { get; set; }
        public string Email { get; set; } 
        public string Telefon { get; set; }
        public string KorisnickoIme { get; set; }
       
        public byte[] Slika { get; set; }
        public int GradId { get; set; }

        //public virtual Grad Grad { get; set; } 
        //public virtual ICollection<Ocjena> Ocjenas { get; set; }
        //public virtual ICollection<Rezervacija> Rezervacijas { get; set; }
    }
}

File: CabinRent/UI/cabinrent_admin/lib/models/klijent.dart
import 'package:json_annotation/json_annotation.dart';

part 'klijent.g.dart';

@JsonSerializable()
class Klijent{
int? klijentId;
String? ime;
String? prezime;
String? email;
String? telefon;
String? korisnickoIme;
String? slika;
int? gradId;
Klijent(this.klijentId,this.ime,this.prezime,this.email,this.telefon,this.korisnickoIme,this.slika,this.gradId);

factory Klijent.fromJson(Map<String,dynamic> json)=>_$KlijentFromJson(json);

Map<String,dynamic> toJson()=>_$KlijentToJson(this);
}
